admin/resources/js/components/classification/CreateClassification.tsx

```diff
@@ -63,12 +64,22 @@ export const CreateClassificationForm: React.FunctionComponent<CreateClassificat
               type="text"
               rules={{ required: errorMessages.required }}
             />
-            <Input
+            <Select
               id="level"
               name="level"
               label="Level: "
-              type="number"
               rules={{ required: errorMessages.required }}
+              options={[
```

Comment:
I would add this to the array of options so that the user has to choose the value so they don't overlook the field:
`{ value: "", label: "Select a level...", disabled: true },`
You'll also want to add a translation string for _Select a level..._